### PR TITLE
ENG-4528 Improve tomcat logs and revert all formats and implementations in use to slf4j and logback

### PR DESCRIPTION
**Pull request by entgigi**

### CODE REVIEW

File: admin-console/pom.xml [hunk 1]
```diff
@@ -187,6 +187,17 @@
...
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>log4j-over-slf4j</artifactId>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-to-slf4j -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-to-slf4j</artifactId>
+            <version>2.20.0</version>
```
Comment (zonia3000) [inline admin-console/pom.xml]: Version should be defined in the parent pom

File: portal-ui/src/test/java/org/entando/entando/aps/servlet/XSSRequestWrapperTest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    @Mock
+    private HttpServletRequest httpServletRequest;
+    @Mock
+    private Validator validator;
+    @Mock
+    private Encoder encoder;
+
+    @BeforeEach
+    private void setUp() {
+        Mockito.reset(httpServletRequest, validator);
```
Comment (zonia3000) [inline portal-ui/src/test/java/org/entando/entando/aps/servlet/XSSRequestWrapperTest.java]: Is this needed?

File: admin-console/pom.xml (hunk 1 above)
Comment (entgigi) [reply admin-console/pom.xml]: moved in the parent pom

File: portal-ui/src/test/java/org/entando/entando/aps/servlet/XSSRequestWrapperTest.java (hunk 2 above)
Comment (entgigi) [reply portal-ui/src/test/java/org/entando/entando/aps/servlet/XSSRequestWrapperTest.java]: removed, tnx!

**Review by eugeniosant — APPROVED** [review]

**Review by zonia3000 — APPROVED** [review]

**Review by pietrangelo — APPROVED** [review]

**Pull request merged**